src/components/LHNOptionsList/LHNOptionsList.tsx

```diff
@@ -69,36 +69,38 @@ function LHNOptionsList({
 
     const emptyLHNSubtitle = useMemo(
         () => (
-            <View>
-                <Text
+            <View style={[styles.alignItemsCenter, styles.flexRow, styles.justifyContentCenter, styles.flexWrap, styles.textAlignCenter]}>
+                <TextBlock
                     color={theme.placeholderText}
-                    style={[styles.textAlignCenter]}
-                >
-                    {translate('common.emptyLHN.subtitleText1')}
-                    <Icon
-                        src={Expensicons.MagnifyingGlass}
-                        width={variables.emptyLHNIconWidth}
-                        height={variables.emptyLHNIconHeight}
-                        small
-                        inline
-                        fill={theme.icon}
-                        additionalStyles={styles.alignItemsCenter}
-                    />
-                    {translate('common.emptyLHN.subtitleText2')}
-                    <Icon
-                        src={Expensicons.Plus}
-                        width={variables.emptyLHNIconWidth}
-                        height={variables.emptyLHNIconHeight}
-                        small
-                        inline
-                        fill={theme.icon}
-                        additionalStyles={styles.alignItemsCenter}
-                    />
-                    {translate('common.emptyLHN.subtitleText3')}
-                </Text>
+                    text={translate('common.emptyLHN.subtitleText1')}
+                />
+                <Icon
+                    src={Expensicons.MagnifyingGlass}
+                    width={variables.emptyLHNIconWidth}
+                    height={variables.emptyLHNIconHeight}
+                    fill={theme.icon}
+                    small
+                    additionalStyles={styles.mh1}
+                />
+                <TextBlock
+                    color={theme.placeholderText}
```

Comment:
Where does `placeholderText` come from? We should just be using our textSupporting color here I would think.

Reply:
Is placeholderText used anywhere else? I would love to remove it if possible, and just use our standard textSupporting styles which use productDark800 as you mentioned. The icons should still use our icon color though, which is productDark700.

Reply:
Can you show me an example of that? I feel like placeholders should also be using textSupporting.

Either way, we can deal with that later, but let's at least make sure this PR is updated to use textSupporting for the text here.